include/nnvm/top/tensor.h

```diff
@@ -251,6 +265,15 @@ struct ClipParam : public dmlc::Parameter<ClipParam> {
   }
 };
 
+struct AroundParam : public dmlc::Parameter<AroundParam> {
+  int mode;
+  DMLC_DECLARE_PARAMETER(AroundParam) {
+    DMLC_DECLARE_FIELD(mode);
```

Comment:
This is a typo? It cause compilation failure. Suggest to mv the ";" to the end of next line.